client/ayon_core/tools/workfiles/models/workfiles.py

```diff
@@ -185,8 +185,6 @@ def get_file_items(self, folder_id, task_id, task_name):
 
         for filename in os.listdir(workdir):
             filepath = os.path.join(workdir, filename)
-            if not os.path.isfile(filepath):
-                continue
```

Comment:
So - Silhouette projects are folders, not files. (see internal private discussion [here](https://discord.com/channels/517362899170230292/1326520751200800838/1326520757613756457))

I'm just not sure whether we _want_ this change? :)

@antirotor @iLLiCiTiT

Reply:
I'm not against but please add comment that explains that we also want to include directories.

Reply:
Is this what you meant? https://github.com/ynput/ayon-core/pull/1070/commits/8db81098af5d15e20607c661ab510c6b6a6ef767